build-tests/localization-plugin-test-01/build.js

```diff
@@ -1,21 +0,0 @@
-const { FileSystem } = require('@rushstack/node-core-library');
-const child_process = require('child_process');
-const path = require('path');
-const process = require('process');
-
-function executeCommand(command) {
-  console.log('---> ' + command);
-  child_process.execSync(command, { stdio: 'inherit' });
-}
-
-// Clean the old build outputs
-console.log(`==> Starting build.js for ${path.basename(process.cwd())}`);
```

Comment:
Frankly `build.js` may be a better choice for a webpack plugin test, to ensure we aren't making Heft-specific assumptions.

I intentionally set up the API Extractor tests without any Rush Stack dependencies in their **package.json** files, specifically to:

1. make sure we try out our own CLI, and 
2. As end user showcases, to illustrate simple easily understandable setups of how to invoke API Extractor.

Not sure how relevant that is to Webpack, but something to consider.